src/yarpdatadumper/main.cpp

```diff
@@ -29,7 +29,11 @@
 #ifdef ADD_VIDEO
     #include <opencv2/opencv.hpp>
     #include <yarp/cv/Cv.h>
-#endif
+#if CV_MAJOR_VERSION >= 3
```

Comment:
@Nicogene I would better indent this inner if-statement wrt the outer if-statement.